src/sentry/web/frontend/msteams_extension_configuration.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+
+
+from sentry import features
+from sentry.utils.signing import unsign
+
+from .integration_extension_configuration import IntegrationExtensionConfigurationView
+
+
+class MsTeamsExtensionConfigurationView(IntegrationExtensionConfigurationView):
+    provider = "msteams"
+    external_provider_key = "msteams"
```

Comment:
it doesn't seem like you are using the `provider` and `external_provider_key` variable

Reply:
@NisanthanNanthakumar It's used in `IntegrationExtensionConfigurationView`